src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -246,4 +295,15 @@ public function setup(): void
 
         $this->autoSetup = false;
     }
+
+    private function getCurrentTimeInMilliseconds(): int
+    {
+        return (int) (microtime(true) * 1000);
+    }
+
+    public function cleanup()
```

Comment:
please mark as internal and add void return like the other methods have

Reply:
Did remove the `@internal` after the review of @chalasr see https://github.com/symfony/symfony/pull/31977#discussion_r334295302 because the whole class is marked as internal.
Should I still add it?

/cc @chalasr

Reply:
Oh I didn't know. Then it's fine.